iterative_constraints.py

```diff
@@ -333,7 +333,7 @@ def print_results(label, results_by_sum_id, type="summary"):
                         # Use classifier label if classifier is loaded, otherwise use the oracle
                         label = (
                             ent["predicted_label"]
-                            if clf_factuality is not None
+                            if clf_factuality is not None and "predicted_label" in ent
```

Comment:
this is just a way to provide a label for non-hallucinated named entities, because now we're not providing a label for them by a classifier, as we're only running the classifier on hallucinated entities. discussed and we'll likely fix later as this is a good potential entry for bugs. While this might introduce a label for intrinsic hallucation (but likely non-hallucinated), we critically don't consider labels at this level during evaluation (only for inference)